apps/web/middleware.ts

```diff
@@ -191,11 +179,6 @@ export const config = {
     // API routes
     "/api/auth/signup",
     "/api/trpc/:path*",
-    // Routes allowed for POST method (matching `POST_METHODS_ALLOWED_APP_ROUTES` array)
-    "/settings/my-account/general",
-    "/settings/developer/webhooks",
-    "/settings/developer/api-keys",
-    "/teams",
```

Comment:
we can simply remove these routes from `config.matcher` and not need to maintain `POST_METHODS_ALLOWED_APP_ROUTES` array